### ISSUE TITLE
chore: add missing alt to network monitor image

### ISSUE THREAD

**Issue by schalkneethling** [mdn/content#20212]
### MDN URL

https://developer.mozilla.org/en-US/docs/learn/performance/measuring_performance/

### What specific section or headline is this issue about?

https://developer.mozilla.org/en-US/docs/learn/performance/measuring_performance#network_tools

### What information was incorrect, unhelpful, or incomplete?

The network-monitor.png image is missing an `alt` attribute.

### What did you expect to see?

The image to have short descriptive alt text

### Do you have any supporting links, references, or citations?

https://docs.google.com/spreadsheets/d/1cDHee6mwDMlY6zANEmg8MjR8tL-wrCS86uawpWZ4XbA/edit#gid=0
